backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/SparkPlanExecApiImpl.scala

```diff
@@ -609,7 +609,9 @@ class SparkPlanExecApiImpl extends SparkPlanExecApi {
     Seq(
       Sig[HLLAdapter](ExpressionNames.APPROX_DISTINCT),
       Sig[UDFExpression](ExpressionNames.UDF_PLACEHOLDER),
-      Sig[NaNvl](ExpressionNames.NANVL))
+      Sig[NaNvl](ExpressionNames.NANVL),
+      Sig[MonotonicallyIncreasingID](ExpressionNames.MONOTONICALLY_INCREASING_ID)
```

Comment:
I think it is better to put this in `ExpressionMappings.scala`, considering it is a common function, not specific to Velox backend.
Then, you may need to add some code in the below to let CH backend use `DefaultValidator` for fallback (Otherwise, I guess CH CI will fail).
https://github.com/apache/incubator-gluten/blob/main/backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala#L178

Reply:
thanks @PHILO-HE for review. sure, let me do the change.